tests/lib/linter/code-path-analysis/code-path.js

```diff
@@ -25,11 +25,13 @@ const linter = new Linter();
 function parseCodePaths(code) {
     const retv = [];
 
-    linter.defineRule("test", () => ({
-        onCodePathStart(codePath) {
-            retv.push(codePath);
-        }
-    }));
+    linter.defineRule("test", {
+        create: () => ({
```

Comment:
@nzakas I agree with consistency, but a few things:

1. There are 200+ instances of `create: context => ({ Program(node) { ... } });`, and this format is used as a shorthand because they return an object. It's too much work to update all these by hand, so I would have to write a codemod to change to `create(context) { return { Program(node) { ... } } }` including adding the explicit return statement. Perhaps we should allow the shorthand style for functions that simply return an object.
2. I enabled the lint rule `object-shorthand: ["error", "always", { "avoidExplicitReturnArrows": true }]` and auto-fixed 100+ instances of the create function to the style you requested. When we're dealing with hundreds of instances of a function call, as we are in this PR, I think it's critical that we use linting/autofixers to enforce any style preference we have, instead of relying on manual edits.

Reply:
Agreed, thanks.